LICENSE

```diff
@@ -1,277 +0,0 @@
-Eclipse Public License - v 2.0
```

Comment:
Hello, Why removing LICENSE file ?

Reply:
Because there is no such file in the generated plugin sample.

Reply:
OK could you add this back ?

Reply:
done